From b08d7e89c9e210e46da655b4cd4a34e5156af4be Mon Sep 17 00:00:00 2001
From: Aurimas Liutikas <aurimas@google.com>
Date: Fri, 1 Feb 2019 01:13:44 +0000
Subject: [PATCH 6/7] Stop generating ids in android namespace.

This change fixes the following error:
[ 88% 15103/17134] AAPT2 link out/target/product/generic_x86/obj/APPS/TeleService_intermediates/package-res.apk
packages/services/Telephony/res/layout/pref_dialog_editpin.xml:25: warn: generated id 'android:id/message' for external package 'android'.

Change-Id: I043294a0768c053c05309d9d7ce1791b8f7ca791
---
 res/layout/pref_dialog_editpin.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/layout/pref_dialog_editpin.xml b/res/layout/pref_dialog_editpin.xml
index a27869075..94cdadfbd 100644
--- a/res/layout/pref_dialog_editpin.xml
+++ b/res/layout/pref_dialog_editpin.xml
@@ -22,7 +22,7 @@
     android:orientation="vertical"
     android:padding="?android:attr/dialogPreferredPadding">
 
-    <TextView android:id="@+android:id/message"
+    <TextView android:id="@android:id/message"
         style="?android:attr/textAppearanceSmall"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-- 
2.17.1

